CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### Improvements
 
 * (all) [#16537](https://github.com/cosmos/cosmos-sdk/pull/16537) Properly propagated `fmt.Errorf` errors and using `errors.New` where appropriate.
+* (math) [#17497](https://github.com/cosmos/cosmos-sdk/pull/17497) Optimize math.Int.Size for values that fit in 53 bits.
```

Comment:
This one isn't needed. The changelog should be under math/CHANGELOG.md

Reply:
Done. Sorry, I missed the module specific CHANGELOG.